src/crimson/os/alienstore/CMakeLists.txt

```diff
@@ -48,10 +48,12 @@ add_library(crimson-alienstore STATIC ${crimson_alien_srcs}
 target_compile_definitions(crimson-alienstore PRIVATE -DWITH_SEASTAR -DWITH_ALIEN)
 target_include_directories(crimson-alienstore PRIVATE
   $<TARGET_PROPERTY:Seastar::seastar,INTERFACE_INCLUDE_DIRECTORIES>)
-target_link_libraries(crimson-alienstore fmt::fmt)
-target_link_libraries(crimson-alienstore kv)
-target_link_libraries(crimson-alienstore heap_profiler)
-target_link_libraries(crimson-alienstore ${BLKID_LIBRARIES})
-target_link_libraries(crimson-alienstore ${UDEV_LIBRARIES})
-target_link_libraries(crimson-alienstore crimson)
-target_link_libraries(crimson-alienstore blk)
+target_link_libraries(crimson-alienstore
```

Comment:
i'd suggest s/build/cmake/ in the title of commit message. because cmake is more specific than "build", which, for instance, could be referencing rpm and deb packaging recipes.